Moesif.Api/Controllers/BaseController.cs

```diff
@@ -63,9 +62,7 @@ internal void ValidateResponse(HttpResponse _response, HttpContext _context)
                         Byte[] bytes = new byte[body.Length];
                         body.Position = 0;
                         body.Read(bytes, 0, (int)body.Length);
-                        string bodyStr = Encoding.UTF8.GetString(bytes);
-                        JObject jsonErr = JObject.Parse(bodyStr);
-                        bodyData = (string)jsonErr["moesif_error"]["msg"];
+                        bodyData = Encoding.UTF8.GetString(bytes);
```

Comment:
`nitpic` should we include whole body only if we can not get error `msg` ?

```
string bodyStr = ""
try
{
   bodyStr = Encoding.UTF8.GetString(bytes);
   JObject jsonErr = JObject.Parse(bodyStr);
   bodyData = (string)jsonErr["moesif_error"]["msg"];
}
catch (Exception) 
{ 
  bodyData = bodyStr;
};
```

Because media upload (like fax etc) body will be large.

Reply:
discussed privately.. media upload should not be in response body of api.. unlikely we will have very large body for calls to moesif api.